docker-compose.yml

```diff
@@ -1,5 +1,16 @@
 version: "3.3"
 services:
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    ports:
+      - 5432:5432
```

Comment:
Replace this line with `- 5432` in order to avoid errors with the port 5432 being used by another process.